packages/graph-explorer/src/core/StateProvider/displayAttribute.ts

```diff
@@ -29,7 +30,12 @@ export function getSortedDisplayAttributes(
       return mapToDisplayAttribute(name, value, textTransform);
     })
     .toArray()
-    .toSorted((a, b) => a.displayLabel.localeCompare(b.displayLabel));
+    .toSorted((a, b) => {
+      // rdfs:label should always be first
+      if (a.name === RDFS_LABEL_URI) return -1;
```

Comment:
nit: you could create a helper method `is_rdf_label` and then you don't need to import RDFS_LABEL_URI